gdx/src/com/badlogic/gdx/graphics/g2d/TextureRegion.java

```diff
@@ -284,4 +284,18 @@ public void scroll (float xAmount, float yAmount) {
 		TextureRegion region = new TextureRegion(texture);
 		return region.split(tileWidth, tileHeight);
 	}
+
+	/** Copies every {@link TextureRegion} in an array into a new array via shallow copy.
+	 * @param regions {@link TextureRegion} array to copy
+	 * @return An array containing shallow copies of all {@link TextureRegion} that were sent in. */
+	public static TextureRegion[] copyRegionArray (TextureRegion[] regions) {
+		TextureRegion[] copy = new TextureRegion[regions.length];
+
+		int len = regions.length;
+		for (int i = 0; i < len; i++) {
+			copy[i] = new TextureRegion(regions[i]);
+		}
+
+		return copy;
+	}

```

Comment:
I'm not sure we should add utility methods like this. There could be many different needs for different apps, just write this code in your app.